cpp/dolfinx/fem/DirichletBC.h

```diff
@@ -49,8 +50,7 @@ namespace fem
 /// V[0] and V[1]. The array[0](i) entry is the DOF index in the space
 /// V[0] and array[1](i) is the correspinding DOF entry in the space
 /// V[1]. The returned dofs are 'unrolled', i.e. block size = 1.
-std::array<Eigen::Array<std::int32_t, Eigen::Dynamic, 1>, 2>
-locate_dofs_topological(
```

Comment:
These must be fixed also in pybind11 wrappers, otherwise they return List().

Reply:
Fixed.